jupyter-extension/jupyterlab_pachyderm/pps_client.py

```diff
@@ -68,7 +68,7 @@ def from_notebook(cls, notebook_path: Union[str, Path]) -> "PpsConfig":
             raise ValueError("field image not set")
 
         requirements = config.get("requirements")
-        if requirements is not None:
+        if requirements is not None and requirements:
```

Comment:
nit: I think we can return `""` as the default value for `get()` in the line above and then simplify this line to `if requirements:"

Reply:
thanks, i think the defaulting is not even necessary but yeah this can be simplified to `if requirements`